ibis/expr/api.py

```diff
@@ -3988,6 +3988,23 @@ def _table_union(left, right, distinct=False):
     return op.to_expr()
 
 
+def _table_intersect(left, right):
+    """
+    Form the table set intersect of two table expressions having identical
+    schemas.
+
+    Parameters
+    ----------
+    right : TableExpr
+
+    Returns
+    -------
+    intersection : TableExpr
+    """
+    op = ops.Intersection(left, right)
+    return op.to_expr()
```

Comment:
```suggestion
    return ops.Intersection(left, right).to_expr()
```

Just a suggestion, feel free to leave it as is. But this seems a bit easier to read to me.

Reply:
I agree that does look a bit nicer

Reply:
@datapythonista Although again, I used the same format as union was written in so should we change that as well to be consistent?

Reply:
yes ok to change both to be consistent